docs/examples/watch-only-wallet.js

```diff
@@ -20,10 +20,10 @@ const bcoin = require('../..');
   // m'/44'
   const bip44Key = priv.derive(44, true);
 
-  // m'/44'/0'
-  const bitcoinKey = bip44Key.derive(0, true);
```

Comment:
Maybe we could modify the examples to say "floKey" just for clarity that it isn't the bitcoin derivation